src/helper/parameter.cpp

```diff
@@ -87,6 +87,27 @@ parameter parameter::string(std::string_view str) noexcept
     return parameter{obj};
 }
 
+parameter parameter::boolean(bool value) noexcept
```

Comment:
Did you check if other types are being generated as strings instead?

Reply:
Do you have an example? I didn't find any

Reply:
There are two overloads to generate strings from integers, have you checked that they aren't being used?

Reply:
Yes, all good